openedx_learning/core/publishing/migrations/0001_initial.py

```diff
@@ -17,360 +19,145 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name="LearningPackage",
+            name='LearningPackage',
             fields=[
-                (
-                    "id",
-                    models.BigAutoField(
-                        auto_created=True,
-                        primary_key=True,
-                        serialize=False,
-                        verbose_name="ID",
-                    ),
-                ),
-                (
-                    "uuid",
-                    models.UUIDField(
-                        default=uuid.uuid4,
-                        editable=False,
-                        unique=True,
-                        verbose_name="UUID",
-                    ),
-                ),
-                ("key", models.CharField(max_length=255)),
-                ("title", models.CharField(max_length=1000)),
-                (
-                    "created",
-                    models.DateTimeField(
-                        validators=[
-                            openedx_learning.lib.validators.validate_utc_datetime
-                        ]
-                    ),
-                ),
-                (
-                    "updated",
-                    models.DateTimeField(
-                        validators=[
-                            openedx_learning.lib.validators.validate_utc_datetime
-                        ]
-                    ),
-                ),
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('uuid', models.UUIDField(default=uuid.uuid4, editable=False, unique=True, verbose_name='UUID')),
+                ('key', openedx_learning.lib.fields.MultiCollationCharField(db_collations={'mysql': 'utf8mb4_bin', 'sqlite': 'BINARY'}, max_length=1000)),
```

Comment:
@bradenmacdonald, @kdmccormick, @feanil: This is how the migration gets written out.